src/components/Common/ProposalPreviewCard/VoteModule.tsx

```diff
@@ -0,0 +1,68 @@
+import { useMemo } from 'react'
+
+import useAuthContext from 'decentraland-gatsby/dist/context/Auth/useAuthContext'
+import isEmpty from 'lodash/isEmpty'
+
+import { ProposalAttributes } from '../../../entities/Proposal/types'
+import { Vote } from '../../../entities/Votes/types'
+import { calculateResult } from '../../../entities/Votes/utils'
+import useFormatMessage from '../../../hooks/useFormatMessage'
+import CategoryPill from '../../Category/CategoryPill'
+import ChevronRight from '../../Icon/ChevronRight'
+
+import './VoteModule.css'
+
+interface Props {
+  proposal: ProposalAttributes
+  votes?: Record<string, Vote>
+}
+
+function VoteModule({ proposal, votes }: Props) {
+  const t = useFormatMessage()
+  const [account] = useAuthContext()
+  const hasVote = !!account && !isEmpty(votes?.[account])
+  const choices = useMemo((): string[] => proposal?.snapshot_proposal?.choices || [], [proposal])
+  const vote = hasVote && !!votes?.[account].choice ? choices[votes?.[account].choice - 1] : undefined
+  const results = useMemo(
+    () => calculateResult(proposal?.snapshot_proposal?.choices || [], votes || {}),
+    [proposal?.snapshot_proposal?.choices, votes]
+  )
+  const vpInFavor = results[0].power || 0
+  const threshold = proposal?.required_to_pass || 0
+  const neededForAcceptance = threshold - vpInFavor
+  const isThresholdStillNotMet = neededForAcceptance >= 0
+  const votingConsensusText = t(
+    `page.home.open_proposals.${isThresholdStillNotMet ? 'threshold_not_met' : 'threshold_met'}`
+  )
+  const votingNeededText = t(`page.home.open_proposals.${isThresholdStillNotMet ? 'vp_needed' : 'vp'}`, {
+    value: t('general.number', { value: isThresholdStillNotMet ? neededForAcceptance : vpInFavor }),
+  })
+  return (
+    <div className="ProposalPreviewCard__Section ProposalPreviewCard__VoteModule">
```

Comment:
Could we move ProposalPreviewCard__Section to a component `ProposalPreviewCardSection` so it can be reused on all components. This new component could receive the `VoteModule` classname.